Kernel/Storage/AHCIPort.cpp

```diff
@@ -55,6 +56,10 @@ AHCIPort::AHCIPort(const AHCIPortHandler& handler, volatile AHCI::PortRegisters&
     dbgln_if(AHCI_DEBUG, "AHCI Port {}: Command list region at {}", representative_port_index(), m_command_list_region->vaddr());
 }
 
+AHCIPort::~AHCIPort()
```

Comment:
Why is the empty destructor needed, isn't the default generated destructor enough?

Reply:
I'd have to include WorkQueue.h in the AHCIPort.h header. I don't really care either way.